src/node/rpc/test/member_voting_test.cpp

```diff
@@ -312,238 +192,6 @@ struct TestNewMember
   crypto::Pem cert;
 };
 
-DOCTEST_TEST_CASE("Add new members until there are 7 then reject")
```

Comment:
This made use of `query` and `read` but I decided to scrap it as it is already covered by end-to-end tests.